files/tinycolor/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "tinycolor",
+  "repo": "bgrins/tinycolor",
+  "files": {
+    "basePath": "tinycolor.js"
```

Comment:
Any chance we can get the min file like we already have here http://www.jsdelivr.com/#!tinycolor ?